Gemfile

```diff
@@ -135,3 +135,5 @@ gem "action-cable-testing"
 gem "icalendar"
 # Easily schedule tasks in ruby <3
 gem "rufus-scheduler"
+
+gem "rqrcode", "~> 1.1"
```

Comment:
keep consistency with the rest of the files by adding a comment above the gem and what it does